bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -889,9 +891,51 @@ class VideoProvider extends Component {
     if (isAbleToAttach) {
       this.attach(peer, video);
       peer.attached = true;
+
+      if (isLocal) {
+        const deviceId = MediaStreamUtils.extractVideoDeviceId(peer.bbbVideoStream.mediaStream);
+        const { type, name } = getSessionVirtualBackgroundInfo(deviceId);
+
+        this.restoreVirtualBackground(peer.bbbVideoStream, type, name).catch(() => {
+          // if it fails, try again after 2 sec
+          setTimeout(() => {
+            this.restoreVirtualBackground(peer.bbbVideoStream, type, name).catch((error) => {
+              this.handleVirtualBgError(error, type, name);
+            });
+          }, 2000);
+        });
+      }
     }
   }
 
+  restoreVirtualBackground(stream, type, name) {
+    return new Promise((resolve, reject) => {
+      if (type !== EFFECT_TYPES.NONE_TYPE) {
+        stream.startVirtualBackground(type, name).then(() => {
+          resolve();
+        }).catch(() => {
+          reject();
```

Comment:
You're rejecting with no error here but using an error reference in L#902 and L#903